src/python/lib/ensembl/compara/config/mlss_species_set_adaptor.py

```diff
@@ -0,0 +1,52 @@
+"""
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from typing import List
+from xml.etree import ElementTree
+import os
+
+def get_species_set_by_name(file: str, name: str) -> List[str]:
+    """Parses mlss_conf.xml file and returns a list of species for a specified species set name.
```

Comment:
I think the code would improve readability if the variables had more meaning on their own:
```suggestion
def get_species_set_by_name(mlss_conf_file: str, species_set_name: str) -> List[str]:
    """Parses `mlss_conf_file` and returns a list of species for the given species set name.
```

Reply:
I was actually in a doubt whether to be very general or more specific. Thanks!